### DISCUSSION TITLE
Mail Signature Configuration

### DISCUSSION THREAD

**Discussion by mirkoesins** [nextgenhealthcare/connect#5446] [Feature/Enhancement Ideas]
Hi,

could you provide to configure mail signatures in the administrator settings and the smpt sender template?
I am sorry if this is possible at the time. The configuration shoud be possible with password and minimum in p12-format.

**Comment by jonbartels** [comment]
Try looking for how to do this using Apache Commons Email. That is the Mail library that Mirth Connect uses.

Anything you can do in the GUI settings you can do using Javascript template steps and Java libraries. For example here is a user using Apache Commons Mail in Mirth to send attachments - https://forums.mirthproject.io/forum/mirth-connect/support/18824-javascript-to-email-attachments

If you can generate the email and signature using Apache Commons Mail then you can port that code to run in Mirth Connect.
